react/Row.tsx

```diff
@@ -147,7 +147,9 @@ const Row: StorefrontFunctionComponent<Props> = ({
                   : `pr${colGap}`
               } ${preventVerticalStretch ? '' : 'items-stretch'} ${
                 preventHorizontalStretch ? '' : styles.stretchChildrenWidth
-              } ${col.width === 'grow' ? 'flex-grow-1' : ''} flex`}
+              } ${col.width === 'grow' ? 'flex-grow-1' : ''} flex ${
+                handles.flexRowItemWrapper
```

Comment:
Could you add this at the beginning at the `className` to make it easier to know the element in the dev tools?